frdm_kw41z: Current master fails compilation in drivers/pwm/pwm_mcux_tpm.c

Build error messages:

```
00:13:06.112  2883 | #define DT_FOREACH_OKAY_INST_nxp_kinetis_tpm(fn) fn(0) fn(1)
00:13:06.112       |                                                        ^~
00:13:06.112 /home/buildslave/workspace/zephyr-upstream/zephyr/include/sys/util_internal.h:106:36: note: in expansion of macro 'DT_FOREACH_OKAY_INST_nxp_kinetis_tpm'
00:13:06.112   106 | #define UTIL_PRIMITIVE_CAT(a, ...) a##__VA_ARGS__
00:13:06.112       |                                    ^
00:13:06.112 /home/buildslave/workspace/zephyr-upstream/zephyr/drivers/pwm/pwm_mcux_tpm.c:203:1: note: in expansion of macro 'DT_INST_FOREACH_STATUS_OKAY'
00:13:06.112   203 | DT_INST_FOREACH_STATUS_OKAY(TPM_DEVICE)
00:13:06.112       | ^~~~~~~~~~~~~~~~~~~~~~~~~~~
00:13:06.112 /home/buildslave/workspace/zephyr-upstream/zephyr/include/device.h:186:2: error: type defaults to 'int' in declaration of 'DEVICE_DT_DEFINE' [-Werror=implicit-int]
```

Full log:
https://ci.linaro.org/view/lite-iot-ci/job/zephyr-upstream/5818/PLATFORM=frdm_kw41z,ZEPHYR_TOOLCHAIN_VARIANT=zephyr,label=docker-xenial-amd64-13/console

Specific commit built: d5072d13786edc4bc34a169e743d5aeeee34a819

--- a/drivers/pwm/pwm_mcux_tpm.c
+++ b/drivers/pwm/pwm_mcux_tpm.c
@@ -194,7 +194,7 @@ static const struct pwm_driver_api mcux_tpm_driver_api = {
 		.mode = kTPM_EdgeAlignedPwm, \
 	}; \
 	static struct mcux_tpm_data mcux_tpm_data_##n; \
-	DEVICE_DT_INST_DEFINE(n, &mcux_tpm_init, device_pm_control_nop \
+	DEVICE_DT_INST_DEFINE(n, &mcux_tpm_init, device_pm_control_nop, \
 			    &mcux_tpm_data_##n, \
 			    &mcux_tpm_config_##n, \
 			    POST_KERNEL, CONFIG_KERNEL_INIT_PRIORITY_DEVICE, \

